addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -265,6 +268,11 @@ func mapAntreaConfigSpec(cluster *clusterv1beta1.Cluster, config *cniv1alpha2.An
 		configSpec.Antrea.AntreaConfigDataValue.Multicast.IGMPQueryInterval = config.Spec.Antrea.AntreaConfigDataValue.Multicast.IGMPQueryInterval
 	}
 
+	if semver.Compare(version, "v1.9.0") >= 0 {
```

Comment:
makes sense to keep the semver.Compare in the controller so the client can use both v1alpha1 = 1.7 AND v1alpha2 schemas 👍